docs/quickstart.rst

```diff
@@ -0,0 +1,52 @@
+HPy Quickstart
+=============
```

Comment:
Doesn't this duplicate much of the [simple example](https://docs.hpyproject.org/en/latest/api.html#a-simple-example)

Reply:
Yes, but I think the two can serve slightly different purposes: quickstart -- when I am impatient, I want hands on experience right now, I don't mind following instructions blindly for now, simple example -- when I want some additional context and explanations. I could not resist adding few explanatory comments to quickstart, but I tried to keep them very short.